setup.py

```diff
@@ -11,14 +11,16 @@ def read(fname):
     return open(os.path.join(os.path.dirname(__file__), fname)).read()
 
 
-setup(name='oemof_visio',
-      version='v0.0.1',
-      author='oemof developing group',
-      author_email='oemof@rl-institut.de',
-      description='The open energy modelling framework',
-      url='https://oemof.org/',
-      namespace_package=['oemof_visio'],
-      long_description=read('README.rst'),
-      packages=find_packages(),
-      package_dir={'oemof_visio': 'oemof_visio'},
-      install_requires=['matplotlib', 'pandas'])
+setup(
+    name="oemof_visio",
+    version="v0.0.1",
+    author="oemof developing group",
+    author_email="oemof@rl-institut.de",
+    description="The open energy modelling framework",
+    url="https://oemof.org/",
+    namespace_package=["oemof_visio"],
+    long_description=read("README.rst"),
+    packages=find_packages(),
+    package_dir={"oemof_visio": "oemof_visio"},
+    install_requires=["matplotlib", "pandas", "graphviz", "oemof.solph"],
```

Comment:
I would use `extra_requires` for graphviz as people do need extra libraries.

Reply:
So one would then `pip install oemof-viso[network-graph]` or  `pip install oemof-viso[graphviz]`?

Reply:
Done in f24a4558d60baa1d68d2857f63f044a78cceac5c